.env.j2

```diff
@@ -1,8 +1,3 @@
 FLASK_APP=todo_app/app
 FLASK_DEBUG=true
 SECRET_KEY=secret-key
-TRELLO_API_TOKEN={{ trello_api_token }}
```

Comment:
(Certainly no harm, but don't feel you need to maintain the ansible files going forward)